packages/dcos-integration-test/extra/test_checks.py

```diff
@@ -0,0 +1,87 @@
+import random
+import uuid
+
+
+__maintainer__ = 'branden'
+__contact__ = 'dcos-cluster-ops@mesosphere.io'
+
+
+def test_checks_cli(dcos_api_session):
```

Comment:
I believe you'll need to add typehint information otherwise the `mypy` will complain.

Reply:
I can't find any tests that add a type hint for the `dcos_api_session` arg, so I'll leave this.

Reply:
We only check typehints on Enterprise DC/OS.